suite-native/module-accounts-import/src/useShowImportError.ts

```diff
@@ -25,46 +20,51 @@ const alertErrorMap: Record<AlertError, AlertErrorOptions> = {
         title: 'Invalid Public address (XPUB)',
         description: 'Check and correct the public address (XPUB).',
         icon: 'warningTriangleLight',
-        isRetryEnabled: false,
     },
     invalidReceiveAddress: {
         title: 'Receive address invalid',
         description: 'Check and correct the receive address.',
         icon: 'warningTriangleLight',
-        isRetryEnabled: false,
     },
     networkError: {
         title: 'Network error',
         icon: 'noConnection',
         description:
             'We were unable to retrieve the data from the blockchain due to a network error.',
-        isRetryEnabled: true,
     },
     unknownError: {
         title: 'Something went wrong',
         icon: 'warningTriangleLight',
         description: 'We are unable to gather the data right now. Please try again.',
-        isRetryEnabled: true,
     },
 };
 
 const pictogramVariant: PictogramVariant = 'red';
 
 export const useShowImportError = (
     networkSymbol: NetworkSymbol,
-    navigation: StackToTabCompositeNavigationProp<
-        AccountsImportStackParamList,
-        AccountsImportStackRoutes.AccountImportLoading,
-        RootStackParamList
-    >,
+    navigation: NavigationProp<any>,
```

Comment:
missing navigation prop

Reply:
It is intentional. So you can send any `NavigationProp` in there.

Reply:
You shouldn't use any navigation prop. It's import error, so the origin will always be AccountsImportStack and you can then type the paths we won't to allow for this hook to redirect. You should always never need to use any for this kind of thing. 

Either make it more flexible by some nice TS generic type, or type it properly but remove `any` please.

Reply:
fixed in c5936dbbb